### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by gulfpearl** [lnbits/lnbits#2165] [Q&A] [answered]
Just curious how the web app implements authentication, I just installed lnbits and it seems if anyone has a user url or any admin, the superadmin url, they can do whatever they want.

**Comment by prusnak** [comment, marked as answer]
> I just installed lnbits and it seems if anyone has a user url or any admin

That is correct. You should not share this URL with anyone.

Other implementation methods are being implemented in PR https://github.com/lnbits/lnbits/pull/2092
